packages/app/src/nativeBridge/modules/mainWindowModule.ts

```diff
@@ -37,7 +38,11 @@ export class MainWindowModule extends NativeBridgeModule {
 
   @moduleFunction({ isRequired: true })
   public async hideToSystemTray(mainWindow: BrowserWindow): Promise<void> {
-    mainWindow.hide();
+    if (platform === 'win32') {
+      mainWindow.hide();
```

Comment:
presumably the app.quit call is the behavior we want on macOS as well? I know they have a tray but it's not quite the same as win

Reply:
lol i've completely forgotten how it worked on mac and i don't have one now... probably safe to follow the quit behavior though